src/pages/_app.js

```diff
@@ -10,7 +11,9 @@ export default function App({ Component, pageProps }) {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
       <AuthProvider>
-        <Component {...pageProps} />
+        <DocumentationEditorProvider>
```

Comment:
question: this needs to be a provider? Could not be just a [custom hook](https://blog.logrocket.com/create-your-own-custom-react-hooks/)?